Removing not needed and incorrect ReferenceCopyLocalPaths references

moozzyk opened the pull request:
Fixes: #142

Commit:
Removing not needed and incorrect ReferenceCopyLocalPaths references

Fixes: #142

moozzyk commented:
/cc @BrennanConroy

BrennanConroy commented on NuGet/Microsoft.AspNet.SignalR.Client.Cpp.WinDesktop.targets.template:
```diff
@@ -47,31 +47,27 @@
       </None>
-      <ReferenceCopyLocalPaths Include="$(MSBuildThisFileDirectory)dll\x64\#Toolset#\Debug\signalrclient.dll" />
     </ItemGroup>
     <ItemGroup Label="Win32 and #Toolset# and Debug" Condition="'$(Platform.ToLower())' == 'Win32' And '$(PlatformToolset.ToLower())' == '#Toolset#' And $(Configuration.ToLower().IndexOf('debug')) &gt; -1">
```
Just curious, how does `Platform.ToLower()` ever equal `Win32`?

moozzyk replied:
wow.. good catch. Looks like the other bug concealed this one. I will try to see what happens and will update the PR...

moozzyk replied:
`.ToLower` is not needed - comparison is case insensitive.

Commit:
Misc fixes to targets file
- removing .ToLower() as MSBuild comparison is case insensitive
- adding a condition to not include debug signalrclient.lib when compiling Release

moozzyk commented:
:up::date: 
found one more issue - in release build both the debug and release versions of signalrclient.lib would be included.

BrennanConroy commented:
:shipit:
